README.md

```diff
@@ -136,82 +288,53 @@ You can use the following channels for getting help with Hazelcast:
   development team and other Hazelcast users.
 * [Stack Overflow](https://stackoverflow.com/tags/hazelcast)
 
-### Using Snapshot Releases
-
-Maven snippet:
-
-```xml
-<repository>
-    <id>sonatype-snapshots</id>
-    <name>Sonatype Snapshot Repository</name>
-    <url>https://oss.sonatype.org/content/repositories/snapshots</url>
-    <releases>
-        <enabled>false</enabled>
-    </releases>
-    <snapshots>
-        <enabled>true</enabled>
-    </snapshots>
-</repository>
-<dependency>
-    <groupId>com.hazelcast</groupId>
-    <artifactId>hazelcast</artifactId>
-    <version>${hazelcast.version}</version>
-</dependency>
-```
 
 ### Building From Source
```

Comment:
Should this be a subsection of "Get Help"? Maybe rather a separate section like "Development" or something else.

Reply:
I added a new contributing section - which was in the Jet readme but missing here.